docs/design.rst

```diff
@@ -1,3 +1,5 @@
+.. _design:
```

Comment:
i can't recall the proper role (maybe also `ref`), but you don't need to explicitly create an anchor inn order to reference documents or sections.